src/praisonai/praisonai/cli/main.py

```
@@ -4063,6 +4063,45 @@ def _run_inline_workflow(self, initial_prompt):
+            return method(*args, **kwargs)
+        except Exception as e:
+            # Handle BudgetExceededError gracefully
+            try:
+                from praisonaiagents.errors import BudgetExceededError
+                is_budget_error = isinstance(e, BudgetExceededError)
+            except ImportError:
+                # Fallback to string check if import fails
+                is_budget_error = 'BudgetExceededError' in str(type(e).__name__)
+            
+            if is_budget_error:
+                from rich import print
```

Ruff (0.15.12)

[error] Import `print` is shadowing a Python builtin

(A004)